forest/oak/base.py

```diff
@@ -555,13 +553,16 @@ def run(study_folder: str, output_folder: str, tz_str: Optional[str] = None,
     from_zone = tz.gettz('UTC')
     to_zone = tz.gettz(tz_str) if tz_str else from_zone
 
+    freq_value = frequency.value
+    freq_str = frequency.name.lower()
```

Comment:
Do these additional variables actually make the code easier to read?

Reply:
they were added for the sake of improving computational time, as they were seen in the profiling of the function as a bottleneck. Ie not extracting info from `enum` and saving it once